Lua/Plugins/py.lua

```diff
@@ -0,0 +1,80 @@
+--- Highlight keywords
+highlight("import", "reserved")
+highlight("from", "reserved")
+highlight("while", "reserved")
+highlight("if", "reserved")
+highlight("else", "reserved")
+highlight("elif", "reserved")
+
+highlight("false", "binary")
+highlight("true", "binary")
```

Comment:
```suggestion
highlight("True", "binary")
```
True should also be capitalized. I forgot this in my previous review :shipit: